playbooks/roles/hammer-devel/tasks/hammer_cli.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: 'Clone the hammer-cli repository'
+  git: repo=https://github.com/theforeman/hammer-cli.git dest=~/hammer-cli update=no
```

Comment:
Nitpick/personal preference for this style syntax for readability and ease of editing:

```
git:
  repo: https://
  dest: ~/hammer-cli
  update: no
```